source/GameInterface/Services/Kingdoms/Messages/RemovePolicy.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Common.Messaging;
+using TaleWorlds.CampaignSystem;
+
+namespace GameInterface.Services.Kingdoms.Messages
+{
+    public record RemovePolicy : IEvent
```

Comment:
Also please add summary comments

```suggestion
    public record RemovePolicy : ICommand
```